### DISCUSSION TITLE
How to install and use on GitHub Actions or Appveyor?

### DISCUSSION THREAD

**Discussion by MarkCallow** [vcsjones/AzureSignTool#177] [Q&A]
The Azure example shows a simple way to install AST. Is there a similar way to install it in a GitHub Actions or Appveyor build using Windows Server 2019 or 2022?

Edited to add the following so as to clean up "suggested" non answer.

Neither vcpkg nor Chocolatey have an Azure Sign Tool Package. Somebody on this project please tell me how to install it on GitHub Actions or Appveyor VMs.

**Comment by MarkCallow** [comment]
Should I take the lack of an answer after almost a month to to mean that no binary packages are available and I have to build it myself?

**Comment by vcsjones** [comment]
Sorry for the delay in response. In your GitHub Action you will just need to make sure .NET 6 is installed and then you can use `dotnet tool install --global AzureSignTool`. Note that AzureSignTool works only on Windows. You can search GitHub for examples of GitHub Actions that do this:

https://github.com/search?q=AzureSignTool+language%3AYAML&type=code&l=Markdown

Here for example, here is Octopus Deploys'. https://github.com/OctopusDeploy/cli/blob/c5e8590ae39f4495ea8d1c7a3bd7f4465fcd5ae6/.github/workflows/go-releaser.yml#L122

**Reply by MarkCallow** [reply]
Thank you for the answer. When you say ".NET 6" is that the .NET framework or the .NET SDK? I ask because I am trying to figure out how to do the same on Appveyor.
